src/app.jsx

```diff
@@ -588,6 +766,14 @@ class Application extends React.Component {
 
     componentWillUnmount() {
         cockpit.removeEventListener("locationchanged", this.onNavigate);
+
+        // Cleanup DBus subscriptions
+        this.state.users.forEach(user => {
+            if (user.dbus) {
+                user.dbus.subscription.remove();
```

Comment:
This might not exist, so I suggest `?` just like you did in the above "streamEvents failed" handler.